MembraneRTC/Sources/MembraneRTC/MembraneRTC.swift

```diff
@@ -161,14 +167,15 @@ public class MembraneRTC: MulticastDelegate<MembraneRTCDelegate>, ObservableObje
  
      - Returns: `LocalCameraVideoTrack` instance that user then can use for things such as front / back camera switch.
      */
-    public func createVideoTrack(videoParameters: VideoParameters, metadata: Metadata) -> LocalVideoTrack {
-        let videoTrack = LocalVideoTrack.create(for: .camera, videoParameters: videoParameters)
+
+    public func createVideoTrack(videoParameters: VideoParameters, metadata: Metadata, simulcastConfig: SimulcastConfig) -> LocalVideoTrack {
```

Comment:
As above, let's document `simulcastConfig`. Please, take a look at other methods too (e.g. `createScreencastTrack`)

Also, looks like we added an empty line between documentation and method definition. Do we have any CI setup for this repo? If not, let's create a Jira story for adding something.